client/src/test/suite/testUtils.ts

```diff
@@ -41,3 +41,11 @@ export async function setTestContent(content: string): Promise<boolean> {
     const all: vscode.Range = new vscode.Range(doc.positionAt(0), doc.positionAt(doc.getText().length));
     return editor.edit(eb => eb.replace(all, content));
 }
+
+export enum Commands {
```

Comment:
Can this be a const enum?

Reply:
Probably. Will fix in subsequent change.